src/elskippy/Archon.java

```diff
@@ -1,9 +1,8 @@
 package elskippy;
 
-import battlecode.common.Direction;
-import battlecode.common.GameActionException;
-import battlecode.common.RobotController;
-import battlecode.common.RobotType;
+import battlecode.common.*;
```

Comment:
hmm.. can you change the setting in intellij to not use wildcards?
otherwise we'll have these random diffs on every damn commit -_-